addons/l10n_fr_pos_cert/report/pos_hash_integrity.py

```diff
@@ -10,10 +10,11 @@ class ReportPosHashIntegrity(models.AbstractModel):
 
     @api.model
     def _get_report_values(self, docids, data=None):
-        if data:
-            data.update(self.env.company._check_pos_hash_integrity())
-        else:
-            data = self.env.company._check_hash_pos_integrity()
+        res = self.env.company._check_pos_hash_integrity()
+        if data and res:
+            data.update(res)
+        if not data:
+            data = res
```

Comment:
This can be simplified

```suggestion
        data = data or {}
        data.update(self.env.company._check_pos_hash_integrity() or {})
```